### DISCUSSION TITLE
Welcome to LearnCPP Discussions!

### DISCUSSION THREAD

**Discussion by Lakhankumawat** [Lakhankumawat/LearnCPP#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bhavneet345** [comment]
I have raised an issue named Diameter of a Binary Tree, can you please tell me where I can commit, or do I need to create a new folder and then commit??

**Comment by Aneesh-07** [comment]
I have created issue #138 , can you please tell me where to include this problem?

**Comment by PrashantVIT1** [comment]
I have created issue #70 , can you please tell me where to include this problem?

**Reply by Lakhankumawat** [reply]
I hope your issue in addressed in PR's

**Reply by PrashantVIT1** [reply]
I had created an issue and it almost done. I have been working on this issue for 14 days and now it is showing me that it has been assigned to [OmarAdelBadawy0](https://github.com/OmarAdelBadawy0) https://github.com/Lakhankumawat/LearnCPP/issues/70.
Kindly look into the cause and please make some important changes as I was about to merge.

**Comment by uhini0201** [comment]
Hi I have raised an issue #78 , can you please tell me whether to include this is dynamic programmming folder or somewhere else??

**Comment by PrashantVIT1** [comment]
I had created an issue and it almost done. I have been working on this issue for 14 days and now it is showing me that it has been assigned to [OmarAdelBadawy0](https://github.com/OmarAdelBadawy0) #70.
Kindly look into the cause and please make some important changes as I was about to merge.

**Comment by dewanshiPaul** [comment]
I have opened issue https://github.com/Lakhankumawat/LearnCPP/issues/802 for a binary search problem, where should i include it?

**Comment by PrashantVIT1** [comment]
I have opened have opened an issue #489 please check the pull request and guide me further.
